streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -721,23 +721,25 @@ public Sensor addLatencySensor(String scopeName, String entityName, String opera
             for (int i = 0; i < tags.length; i += 2)
                 tagMap.put(tags[i], tags[i + 1]);
 
+            String metricGroupName = "streaming-" + scopeName + "-metrics";
```

Comment:
I have thought about it before, the reason I would prefer it to end with the groupName is to distinguish the top-level metrics (with groupName "streaming-metrics") that owned by the framework with the second-level metrics (with groupName "streaming-scope-metrics") that can be created by the users. And if scope is already part of the groupName, it is not necessary to be in the tags any more.